src/commands/download.yml

```diff
@@ -16,6 +16,10 @@ steps:
       working_directory: '~/'
       command: |
         set -e
+        if [ << parameters.redmine_version >> = trunk ]; then
+          svn co --non-interactive --trust-server-cert https://svn.redmine.org/redmine/trunk ~/redmine-trunk
```

Comment:
📝 I tried `svn co` without `--trust-server-cert` and got the following error:

```
$ svn co --non-interactive https://svn.redmine.org/redmine/trunk ~/redmine-trunk
svn: E230001: Unable to connect to a repository at URL 'https://svn.redmine.org/redmine/trunk'
svn: E230001: Server SSL certificate verification failed: issuer is not trusted
```

Probably it is the same issue reported in https://www.redmine.org/issues/24864.

I think installing the missing intermediate certificates is safer but maybe too troublesome?

Reply:
I've found some ways to fetch and use the intermediate certificate, but they look a little lengthy.

```
# Installing with update-ca-certificates
sudo curl -L -o /usr/local/share/ca-certificates/GandiStandardSSLCA2.crt https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem
sudo update-ca-certificates
svn co --non-interactive https://svn.redmine.org/redmine/trunk /tmp/redmine # ok
```

```
# Enabling by Subversion configuraion
curl -L -o /tmp/GandiStandardSSLCA2.pem https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem
cat <<HERE >> ~/.subversion/servers
[global]
ssl-authority-files=/tmp/GandiStandardSSLCA2.pem
HERE
svn co --non-interactive https://svn.redmine.org/redmine/trunk /tmp/redmine # ok
```

Newer versions of Subversion has `--trust-server-cert-failures unknown-ca` option but we have to update Subversion as one on the CircleCI image is a bit older.

Reply:
Thank you for investigating that!

> I think installing the missing intermediate certificates is safer than loosening validation but it may be too troublesome?

I agree with you.
I found an CLI option to pass a config to `svn co`, with `--config-option` we can do the same thing in one-liner.
```
svn co --non-interactive --config-option servers:global:ssl-authority-files=<(curl -sL https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem) https://svn.redmine.org/redmine/trunk /tmp/redmine
```

I have two concerns

- Is the https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem is officially supported? I couldn't find the documentation about that.
- What if svn.redmine.org changes its certificate issuer?

Reply:
>```
>svn co --non-interactive --config-option servers:global:ssl-authority-files=<(curl -sL https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem) https://svn.redmine.org/redmine/trunk /tmp/redmine
>```

Looks cool 😎 

> - Is the https://www.gandi.net/static/CAs/GandiStandardSSLCA2.pem is officially supported? I couldn't find the documentation about that.

Sorry for my negligence. The pem URL is published in https://docs.gandi.net/en/ssl/documents/technical_documents.html.

> - What if svn.redmine.org changes its certificate issuer?

- Case A: If svn.redmine.org doesn't require Gandi's certificate anymore:
    - Jobs will success ✅
    - though a needless certificate is installed.
- Case B: If svn.redmine.org requires another intermediate certificate:
    - Jobs will fail 💥
    - and we have to fix the orb and rerun the jobs.

I think Case A is not a problem as long as Gandi's certificate is trustworthy. Case B will give us some trouble for securing connection.

But now I'm starting wondering how much is it dangerous if the connection with svn.redmine.org in insecure...

Reply:
Thank you! I updated the code.

I test published dev version [`agileware-jp/redmine-plugin@dev:d68d439`](https://circleci.com/gh/agileware-jp/redmine-plugin-orb/71) in my private repository, it looks work well.